### DISCUSSION TITLE
white dwarf convection parameters

### DISCUSSION THREAD

**Discussion by HamidGadirov** [AMReX-Astro/MAESTROeX#472] [Q&A]
Hi,

I am working with white dwarf convection simulation (https://github.com/AMReX-Astro/MAESTROeX/tree/main/Exec/science/wdconvect) and wanted to ask which parameters can be changed within simulation so that to generate an ensemble (runs with different parameters so that to have variations in data).

I am especially interested in density and velocity fields. Is it possible to change parameters so that the data generated be the simulation is a bit different? If that is not possible, could you recommend other simulations from MAESTOeX?

Thanks!
Hamid

**Comment by zingale** [comment]
you could just change the parameters for the initial perturbation,

```
problem.velpert_amplitude
problem.velpert_scale
problem.velpert_steep
problem.velpert_radius
```

small variations on those will be enough to generate different realizations of the convective velocity field.

In fact, the problem is so non-linear, that just running on different number of processors can change the roundoff error from reductions across all processors that you'll generate differences after a while.

**Reply by HamidGadirov** [reply]
Thank you for you quick reply!

Do I understand correctly that these differences in velocity field will affect the density field that is only available for this simulation?

**Reply by biboyd** [reply]
@HamidGadirov To change the density field you need to use a different initial model by setting the `maestro.model_file` parameter. There are some models in the `wdconvect/model_files/`. 

You can also generate parameterized models with the code  in https://github.com/AMReX-Astro/initial_models/tree/main/spherical. There are also a few examples in that directory of different central densities and temperatures.

**Reply by HamidGadirov** [reply]
@biboyd I recently checked the density for the different models from `wdconvect/model_files/`, however it doesn't change during the simulation. I am talking about the `('boxlib', 'rho')` variable from the simulation. There are many other variables, and I liked how the velocity changes, so I can definitely use it for my ML-based experiments. I wanted to ask, which other fields can you recommend to use from the wdconvect simulation? Since I already have the velocity (vector) field, I am interested in a scalar field. I see that there is `('boxlib', 'entropy')` and `('boxlib', 'thermal')` (is it thermal energy?), perhaps one of these?

Also, could you help me to understand the velocity field? What are we looking at there? Is it the velocity of the density field? Then why there are no changes in the density?
